stories/ImageView.stories.tsx

```diff
@@ -0,0 +1,44 @@
MSuNAFMEIyCDkwwQhI8AEPXJIEJSjBBzyZTGSegAI3oIAcRdABEkzAAlewQwK6sY13dNMd7/gGL1ghCl6IwhOG8MQuvuGNTRAhEF0MBBS00AUq+CAHMf6wpAg8QAIQGLMEK/AkCDzpgRFQYAPQvEElY4CBImBBm9twRzy04Y13XEMWnggFK0zDikYcIp2o2EQBuHDAR0ABgjvMZzJL4AEUwGAFNYgpS0mwz4KOYAMKVSgPcuADKTQhSgl4h2u+2QtGrCETv2gFLGBhjFM4Yg1KIEQB4vAILjxCCjoYogm2utUSiKAEMGDBClggUw94YKAlGIFCcXoDKXQhC1lowxLoAIigjmsh7WBEHhABiUhAQn57GEQdRgWHMFSBClOwpw8smVYRjMCxK1XBV1eaTBV4Na0U+MMGMmAHPdgCDwpTwwIA0Zpt8A8bhqBDHfawhzKUIQxlIP5DHrdAhTAg1gc26EEN0poCEfQ2BSmgbGQtK1yD+iEJhLhFLepRjbV4IgEJ2IVQVzKNO9ihDlsgAxVsYIMfUKEJUYjCdmvA3RoE1KsiSK8KelsC4gr3q5a1rFod+Yt1CKsh+NAGK2ShjVvlIxlqQEMbxAsDFFwSBjCQQQVtUAMEI7O9I5Bseh872coKl7Im6IMGCvENfJRjIdXAB6wk0w9nuGQXZ1ACFnRQAhB2oAMfqAAGMIDgD8CgmcnsrW/TG1wLX3ilIzjBHwywCW/0Yx6Y4V885CEPickjHro4QxKy8AI61rEDFcikJjWZzPNOOAWSTeZX4dveC79gyLjwRv5O3uYPbIiDHhESKiuUsIQskEACV+5ACOD4AU1u0gQBFYGEd1xmH1d2BCvNgSQM8Ix3zCNF+oAQPcYxjnh4Y867PEEEQtgBD4jgBEP8AAjUWIIYrCCgKXishHv84/fmIBIGKEY2mtEPfcwjQkOL8zuMoQQiZAEIpXwACBxrghNIYAJ9XiYJTn1qFTj72Spg9gomWYMe6Na8p87BBgBxim4shH8CecY+mjHpSr9jF2kwwA6CIAEWpAAFHuiABOadbGYCNNrQjjZAxSyCEEib2THoQiFCYQ18iEQe7tBWk+McD2GsoQAZyEEJvsACFqBAzy6IQZ832eVmt/fZYhDDF/6+8ANr96DjAV1BDhyZiF/UCh+y8obMleEXeeRDG40oQAEwwAMPVPwLMdDBAx3MyRsDdAX5VoG1f1Byk0cW4DvQwBIuMYz7aeMaw9BEK9SQjP2hIx/vcMURLHCBGYCgBmEYwxRmgAEazADBCF5ps/MdgbpLIARf/ffRTbCDG2DADZYAxi1gIYhAGCANBwiGwR8dj2AggggXoIEN0n6+8pW9vCsw+tyhfWprfyHkY7A2D3CAbRWAQQg6eBwf6oAEAyQhCYEggjTWwQ7+4YMaqMCCBSbQgjDYtgUXcML5uHtGzTs7ws8egx/GwHzljyHkJMf2CHAABjHQwKFNsIAMeP75CCIMgxvRADc7ZpGHIQxgCEUowhCGgIZK5GIL4o2g5k+NA0DnYAZUoIIOcmd5Mvi/Cj8QaBGAA0BwQbIjAxUgBBMABQBQdetwD2vmJbwgCwegAGrgCbRADfHgDrBwBG3gBBdgBXjXAxEgBi+AAZLABEmABoKQCsAgDdYgDcLgCGcwBHvwCDXgAT2wbjsQTOhVAhUgA1eAANcgc/ewEgJxD+nADEd4D/lwhPkAD7uQAGlwBdvTASKAAxIgBBgABYGgBIvQCrtgDVYhEevADbOgCINABf2UAjvwhjlAYYIWAlIwhN7QDa6hJQWhD/AwCx4IBRgwBR0AXFlYAYVjAP658C3uQEgCAXNZRAg0UALAFQN9ZwLphQIowAIfIIR7tA7NsxHnkA7oMA9OeBC7gABYkAQTAARm5QGpJgHeUwDF0A1ZAQ9TcQ/wwA3dMAyBgAE/MAYscAI3cAIUhok4UAJNgAUIsA3bgCVJeIR44g7GQARX0AQT8AUggIkokAIc4Ac6AADFsA0pUQ6jSEi84A286AIh51gjMALA9W4o4AUw8AROMADbQA3O2A9NiA5IOBL84A/3sA6wYAERFAFeoI2Z2AFVEAQZkAvcQCxIBm6+oA3EEAgaMAUs8ADt6GwpYHEsEAb0WI/bQHtMcYTncJLogA70cA/b8AoWEAUz0P4B2BiPIOAFHmACYrABhVAN81AOSLaS8vAN36AKbRCIJIgDzoYDXrCUXgCSlDMA1GAO7FCKHXEP7YAO2KAKLykDEvAD2egFDwAGKVAB3rME1QAOaAkO4cAMzbANu5AISlAEVVAGX3ADOHCXTBkGe0CPZjAAx8IOHPGER5iS5zANqpAEGdBLHwABIMRPJqBtG2AAtoAM4VCZ4bAMy6AMrAAITIAFWTADEQAEQSAEP0ByT0AFe/AGZmAEehANxMIR+nCE8oAOMoENoaABBoA1TdAEYeB2UmAFXcAETNAHnXAMlFmZy8AMpNAIaZAGbvBXW2AF0pl/W1CdlGMErBkNJv7BEftoFIWpCoCAAErgBkhQBG/QBEUABVAgnICwAA1QnMiAmc2gDKVwCAtwBmhgB2/QBq/3ek5QBEiABNhpBHjwCdgQK9w5mDLhDNggDI2gBmpwBgkwAEaABEegBBiqBEzgnvDpDJipDKPQCaJlBqs5AEQAXUQwoEZAop0FGe8gDwl6D0h2kueADbpQCo1gCHeAB3lwBz56B4AQpIDQCaSwDOIgDh6qDL5wo4xwNiQapED6o3fAGKEQC9jgDeqQDXiCEPeQkkahDs+gDMqQDMlQDdVQC7QQC7LACrrQpvGJDupADs4wp9gADclQC7aAp7YQC7qAGLTgMbRQC7WQDHfZUKjZMCcJmpJxSg7qQA+kqA+akg7YMKnPkA3q4Azp0KjZ8AyVmg3YYA6gGqqTyqDYMA3lMA3UoAzYIA/+UKiIGqP0QA70MKtIVg7gUA9dmqmW6g/O0KhgWqnqoA7tEJXVEKpHepLgUA7Kag/TgA35IA+FKg8BAQA7',
```

Comment:
Could we put the files into a `fixtures` folder and `fs.readFileSync` them instead of hardcoding a long string like this?

Reply:
Sure, that sounds like a good idea. Would those fixtures live next to the component, or with the stories?

Reply:
Probably in `stories/` if you're not using any of these files in the test file of the `ImageView` component, WDYT?

Reply:
We discussed this in IRC and there doesn't seem to be a way to make this work.